cmd/cluster/dynatrace/logsCmd.go

```diff
@@ -76,7 +113,14 @@ func main(clusterID string) error {
 	}
 
 	fmt.Println(query.Build())
-	fmt.Println("\nLink to Web Console - \n", GetLinkToWebConsole(hcpCluster.DynatraceURL, since, base64.StdEncoding.EncodeToString([]byte(query.finalQuery))))
+
+	url := GetLinkToWebConsole(hcpCluster.DynatraceURL, since, query.finalQuery)
+
+	if strings.Contains(url, "failed to create JSON for sharable URL") {
+		return fmt.Errorf("failed to get url")
+	}
```

Comment:
We should properly return errors and bubble them up, instead of just returning a single value

Reply:
Added return error for _GetLinkToWebConsole_ func to resolve above review